tests/test_utils.py

```diff
@@ -215,3 +217,10 @@ def test_intersection():
     inters3 = intersection(multi6, multi7)
     expected_inters = MultiPolygon([Polygon([(1, 0), (1, 1), (2, 1), (2, 0)])])
     compare_multipolygons(inters3, expected_inters)
+
+
+def test_enumerate_bands():
+    assert enumerate_bands(1) == ["A"]
+    assert enumerate_bands(26) == [x for x in string.ascii_uppercase]
```

Comment:
This assert doesn't say anything about the band enums actually being unique. This assert could pass if there were 26 "A"s, right?

Reply:
Well, indirectly it does! This specifically asserts that the result is equal to the list ["A", "B", ... "Z"]. I KNOW that string.ascii_uppercase is a list of unique characters. But this is a good point, I'll add one that checks for uniqueness in a >26 band situation. Thanks!